scipy/stats/tests/test_morestats.py

```diff
@@ -1919,48 +1919,52 @@ def test_brack(self):
                             -0.71215366521264145, decimal=7)
 
 
+@skip_xp_backends(cpu_only=True)
+@array_api_compatible
 class TestBoxcox_llf:
 
-    def test_basic(self):
+    def test_basic(self, xp):
```

Comment:
Looks like there are no tests of native dtype or `float32`. What do you think of `parametriz`ing this one over those possibilities?

Reply:
Looks like we need some tol changes

Reply:
I think it's just that somewhere in the `float32` calculation the result is converted to an inaccurate `float64`. The reference value is still `float64`, so the test didn't fail due to `dtype`. So if the `dtype` is not promoted accidentally, I don't think there will be a problem.